projects/plugins/jetpack/_inc/lib/class-jetpack-google-drive-helper.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Google Drive helper.
+ *
+ * @package automattic/jetpack
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Status\Visitor;
+
+/**
+ * Class Jetpack_Google_Drive_Helper
+ */
+class Jetpack_Google_Drive_Helper {
+	/**
+	 * Checks if the user has a valid connection to Google Drive
+	 *
+	 * @param int $user_id The user ID.
+	 * @return array       Array with single 'valid' (bool) entry.
+	 */
+	public static function validate_connection( $user_id ) {
```

Comment:
I'd argue this should be renamed to `has_valid_connection` and always return a boolean.